helm_deploy/templates/ingress.yaml

```diff
@@ -59,4 +59,16 @@ spec:
               {{- end }}
           {{- end }}
     {{- end }}
+    {{- if .Values.metabase.enabled }}
```

Comment:
I think this is inside the ingress enabled option - is this what we want? I imagine we need to be able to see the metabase instance even if we have no ingress switched off for the app.

Reply:
I think it might make more sense if we put this bit in it's own ingress file within the metabase templates folder.